openquake/commonlib/oqvalidation.py

```diff
@@ -75,9 +75,8 @@ class OqParam(valid.ParamSet):
     intensity_measure_types_and_levels = valid.Param(
         valid.intensity_measure_types_and_levels, None)
     # hazard_imtls = valid.Param(valid.intensity_measure_types_and_levels, {})
-    hazard_investigation_time = valid.Param(valid.positivefloat, None)
     interest_rate = valid.Param(valid.positivefloat)
-    investigation_time = valid.Param(valid.positivefloat, None)
+    investigation_time = valid.Param(valid.positivefloat, 50.)
```

Comment:
The missing `investigation_time` error should be raised only if the calculation mode is not scenario. Using a default value when the user forgets to include this parameter in the job file could be misleading.